setup.cfg

```diff
@@ -1,5 +1,5 @@
 [egg_info]
-tag_build = dev
+tag_build =
```

Comment:
Does this need to be here at all?  Should we replace with `production` or something like that?

Reply:
Whatever we put there will be glued onto the end of our package version number so to do a production release we want nothing.  Dropping that line entirely would probably also produce the same result by default, I just imitated what I found in some instructions online.  Seems explicit and harmless, if redundant.  Can remove the line if you prefer.